keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -0,0 +1,395 @@
+the ImageMaximumGCAge may not be exactly precise. An image could be GC'd up to 5 minutes after it has aged out.
+
+Finally, Kubelet restarts are a point that needs to be figured out. The easiest way to handle it would be waiting the full ImageGCMaximumAge for an image to be qualified for GC,
+but that would essentially disable the feature if the Kubelet restarts more frequently than ImageGCMaximumAge.
+
+### Test Plan
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+- `pkg/kubelet/images`: `2023-09-14` - `84.2`
+
+Additional tests will be added to pkg/kubelet/images to unit test the new field and verify it works along with the other GC options.
+
+##### e2e tests
+
+- `test/e2e_node/garbage_collector_test.go`
+
+Additional tests will be added to this file to cover the garbage collection e2e.
+
+### Graduation Criteria
+
+
+#### Alpha
+
+- Configuration field added to the Kubelet (disabled by default)
+- Feature supported by Kubelet Image Manager
+- Unit tests and e2e tests added
+- Add a metric `kubelet_image_garbage_collected_total` which tracks the number of images the kubelet is GC'ing through any mechanism.
+
+#### Beta
+
+- Gather feedback from users
+
+#### GA
+
+- Addition of conformance tests
+- Some examples of real-world usage
+- Allowing time for feedback
+
+### Upgrade / Downgrade Strategy
+
+This option is purely contained within the Kubelet, so the only concern is the flag is added to the configuration of the newer
+Kubelet and then downgraded.
+
+There's nothing the Kubernetes community can do to prevent this, and admins should ensure their configuration fields will function with
+the processes they run.
+
+### Version Skew Strategy
+
+Version skew is not a worry assuming the internal Kubelet changes are synchronized with the configuration changes.
+
+## Production Readiness Review Questionnaire
+
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: ImageGCMaximumAge
+  - Components depending on the feature gate: kubelet
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+
+###### Does enabling the feature change any default behavior?
+
+Yes, when the configuration option is set images will potentially be garbage collected earlier.
```

Comment:
that's not really a change in default behavior; you have to configure the option for it to happen. so "no" would be a better answer

Reply:
updated!